toolchain/sem_ir/import_ir.h

```diff
@@ -14,18 +14,20 @@ namespace Carbon::SemIR {
 // A reference to an imported IR.
 struct ImportIR : public Printable<ImportIR> {
   auto Print(llvm::raw_ostream& out) const -> void {
-    out << "{node_id: " << node_id
+    out << "{decl_id: " << decl_id
         << ", is_export: " << (is_export ? "true" : "false") << "}";
   }
 
-  // The node ID for the import.
-  Parse::ImportDeclId node_id;
-  // The imported IR.
-  const File* sem_ir;
+  // The `import` declaration.
+  InstId decl_id;
   // True if this is part of an `export import`.
   bool is_export;
+  // The imported IR.
+  const File* sem_ir;
 };
 
+static_assert(sizeof(ImportIR) == 16, "Unexpected size");
```

Comment:
The size will be 12 for a 32-bit compilation. If it's worth checking this, maybe `sizeof(ImportIR) == 8 + sizeof(void*)`.

Reply:
Done (though `uintptr_t` I think is supposed to be preferred now?).

I did feel it's worth checking because it had been 24 before the re-pack (this is why I reordered fields).